### PR TITLE
fix: npm registry scoped permission error

### PR DESCRIPTION
**Pull request by dead-horse**
This change is [Reviewable](https://reviewable.io/reviews/cnpm/npminstall/118)

### CODE REVIEW

**Comment by dead-horse** [comment]
strongloop 6.0.1 安装有问题，发了新版本修复了

**Comment by fengmk2** [comment]
我重跑一下

File: lib/download/npm.js [hunk 1]
```diff
@@ -103,6 +114,41 @@ function getPackageNpmUri(pkg, options) {
...
+    dataType: 'json',
+    timeout: options.timeout,
+    followRedirect: true,
+    gzip: true,
+  }, options);
+  options.totalJSONSize += result.res.size;
+  options.totalJSONCount += 1;
+  const all = result.data;
+
+  // fetch with range won't get 401, only need to match tag and version
```
Comment (fengmk2) [inline lib/download/npm.js]: 真神奇

**Comment by fengmk2** [comment]
+1

**Pull request merged**